src/Components/ProvisioningWizard/steps/ReservationProgress/index.js

```diff
@@ -105,11 +107,13 @@ const ReservationProgress = ({ setLaunchSuccess }) => {
     refetchIntervalInBackground: true,
   });
 
-  // error handling
+  // errors and warnings handling
   React.useEffect(() => {
     if (currentInterval === false) {
-      setCurrentError('Session timed out, the reservation took too long to fulfill');
-      return;
+      setCurrentWarning(
+        `The launch progress is slower than expected, but we are still on it.
```

Comment:
When the notification integration will be ready, we can update this warning message